modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -5247,6 +5309,41 @@ private void _assertFilteredObjectEntriesSize(
 			objectEntries.toString(), size, objectEntries.size());
 	}
 
+	private void _assertListTypeEntries(
+		List<Object> expectedListTypeEntries,
+		List<ListEntry> actualListTypeEntries) {
+
+		if (ListUtil.isEmpty(expectedListTypeEntries)) {
+			return;
+		}
+
+		for (Object expectedListTypeEntry : expectedListTypeEntries) {
+			Assert.assertTrue(
+				ListUtil.exists(
+					actualListTypeEntries,
+					listEntry -> {
```

Comment:
Can you rename `listEntry` to `actualListTypeEntry` ?

Reply:
Done!